bc_obps/reporting/tests/service/test_reporting_dashboard_service.py

```diff
@@ -465,3 +503,93 @@ def test_reports_sorting_and_filtering(
             (testing_version[1], operation2.name, last_year),
             (testing_version[2], operation2.name, current_year),
         ]
+
+    def test_report_retrieval_after_transfer(self):
+        ## ARRANGE
+        reporting_year_2024 = ReportingYear.objects.get(reporting_year=2024)
+        reporting_year_2025 = ReportingYear.objects.get(reporting_year=2025)
+
+        # Create original user & operator, grant them access
+        original_user_operator = baker.make_recipe('registration.tests.utils.approved_user_operator')
+
+        # Create new user & operator, grant them access
+        new_user_operator = baker.make_recipe('registration.tests.utils.approved_user_operator')
+
+        operation = baker.make_recipe(
+            'registration.tests.utils.operation',
+            operator=original_user_operator.operator,
+            status=Operation.Statuses.REGISTERED,
+        )
+        timeline1 = OperationDesignatedOperatorTimeline.objects.create(
+            operation=operation,
+            operator=original_user_operator.operator,
+            start_date="2011-09-01T13:20:30+03:00",
+            end_date=None,
+        )
+
+        # make a bunch of other random operations for other operators, so that we can later check that the user doesn't have access to these random operations
+        operation_baker(_quantity=20)
+
+        # create report for operation, filed by original_operator
+        report_r1v1_id = ReportService.create_report(operation.id, reporting_year_2024.reporting_year)
```

Comment:
better not to use bakers

Reply:
I still think it's better to use bakers, but I'm going to leave as-is because it does work and setting up the bakers manually requires a good understanding of reporting